src/index.js

```diff
@@ -40,6 +46,16 @@ async function findOnOtherSites(info, hashesToExclude) {
 	return successful.length;
 }
 
+async function updateSearchTimestamps(infoHash) {
+	const cacheKey = CACHE_PREFIX_TIMESTAMPS + "|" + infoHash;
+	const existingTimestamps = get(cacheKey);
+	const firstSearched = existingTimestamps
+		? existingTimestamps.firstSeen
+		: new Date();
+	const lastSearched = new Date();
+	save(cacheKey, { firstSearched, lastSearched });
```

Comment:
I think `existingTimestamps.firstSeen` should be `existingTimestamps.firstSearched`